src/docfx/config/Config.cs

```diff
@@ -105,18 +105,13 @@ internal sealed class Config
         /// <summary>
         /// Gets the authorization keys for required resources access
         /// </summary>
-        public readonly HttpConfig Http = new HttpConfig();
```

Comment:
also update the spec [https://github.com/dotnet/docfx/blob/v3/docs/designs/credential.md](https://github.com/dotnet/docfx/blob/v3/docs/designs/credential.md)